docs/pages/desktop-access/active-directory.mdx

```diff
@@ -0,0 +1,216 @@
+security as a priority, and may not be as performant as standard RDP clients.
+Consider Desktop Access to manage access to your most sensitive Windows
+environments, not as a drop-in replacement for other tools to provide general
+access to Windows desktops.
+
+</Details>
+
+## Prerequisites
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+- One or more hosts to run the Teleport Auth and Proxy services on.
+</ScopedBlock>
+- A server or virtual machine running a Windows Server operating system.
+  In this guide, we'll install Active Directory on this server in order
+  to support passwordless logins with Teleport to the Windows desktops
+  in the Active Directory domain.
+- A Linux host where you will run the Teleport Desktop Service.
+  <ScopedBlock scope={["oss", "enterprise"]}> This guide assumes that you will
+  run Teleport's Windows Desktop Service on a dedicated host. To install Desktop
+  Access into an existing Teleport instance running other services, see the
+  [Manual Setup](active-directory-manual.mdx) guide for Desktop Access.</ScopedBlock>
+- An Active Directory domain, configured for LDAPS (Teleport requires an
+  encrypted LDAP connection). Typically this means installing
+  [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
+
+## Step 1/2. Install Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/2. Run the discovery wizard
+
+In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>
+`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>
+`mytennant.teleport.sh`</ScopedBlock>. Click on your user name at the top right
+and select **Manage Access**, Select "Desktop" from the main menu, then **NEXT**:
+
+If you already have Active Directory installed, skip to the next step. Otherwise,
+copy and paste the first command provided into a Windows PowerShell window. If
+you aren't already running AD Certificate services, copy and paste the second
+command after the first one completes and the server restarts:
+
+![Install Active Directory](../../img/desktop-access/install-active-directory.png)
+
+Once the server is restarted from one or both command executions, click **NEXT**.
+
+Copy and paste the provided command into a Windows PowerShell window to download
+and run the 'configure Active Directory' script:
+
+![Configure Active Directory](../../img/desktop-access/configure-active-directory.png)
+
+Click **NEXT**.
+
+The PowerShell script will output a Teleport configuration block. Copy this
+block to a temporary location. Click **Next**.
+
+On the Linux host you installed Teleport to run as the Desktop Access connector,
+edit `/etc/teleport.yaml` and paste the configuration provided by the output of
+the previous step. Review and edit the addition as necessarily for your environment:
+
+```yaml
+version: v3
+teleport:
+  auth_token: 2239...c5b21
```

Comment:
This renders in two different colors in the code snippet. Should we wrap it in quotes or use a variable?

Reply:
Replaced with a token preset var.